apps/lcs/behaviours/send-email-notification.js

```diff
@@ -39,10 +39,25 @@ const getPersonalisation = (recipientType, req) => {
     landlord_company: req.sessionModel.get('landlord-or-agent-company'),
     landlord_email: req.sessionModel.get('landlord-or-agent-email'),
     landlord_tel: req.sessionModel.get('landlord-or-agent-tel'),
-    rental_postcode: req.sessionModel.get('valuesEnriched').tenantType === 'Prospective tenant' ?
+    prospective_rental_postcode: req.sessionModel.get('valuesEnriched').tenantType === 'Prospective tenant' ?
       req.sessionModel.get('rental-property-postcode') : '',
     rental_property_address: req.sessionModel.get('valuesEnriched').tenantType === 'Tenant' ?
-      req.sessionModel.get('tenantAddress').join('\n') : ''
+      req.sessionModel.get('tenantAddress').join('\n') : '',
```

Comment:
Should be joined with `(', ')` to match the figma format. Same for the `full_address` param above. Also not certain this copes with optional fields address line 2 and county too well. However, I think that can be dealt with in the PR for TLF-117